app/actions/loginActions.js

```diff
@@ -0,0 +1,59 @@
+import { push } from 'react-router-redux';
+import { LOGIN_REQUEST, LOGIN_SUCCESS, LOGIN_ERROR } from './actionTypes';
+
+function loginRequest () {
+  return {
+    type: LOGIN_REQUEST,
+    payload: {
+      loggingin: true
```

Comment:
Minor suggestion: If we're keeping the camelCase convention, this should be `loggingIn`.